web-services-app/src/main/java/edu/unc/lib/boxc/web/services/processing/BulkRefIdCsvExporter.java

```diff
@@ -0,0 +1,125 @@
+package edu.unc.lib.boxc.web.services.processing;
+
+import edu.unc.lib.boxc.auth.api.Permission;
+import edu.unc.lib.boxc.auth.api.exceptions.AccessRestrictionException;
+import edu.unc.lib.boxc.auth.api.models.AgentPrincipals;
+import edu.unc.lib.boxc.auth.api.services.AccessControlService;
+import edu.unc.lib.boxc.model.api.ResourceType;
+import edu.unc.lib.boxc.model.api.exceptions.InvalidOperationForObjectType;
+import edu.unc.lib.boxc.model.api.exceptions.RepositoryException;
+import edu.unc.lib.boxc.model.api.ids.PID;
+import edu.unc.lib.boxc.model.api.objects.WorkObject;
+import edu.unc.lib.boxc.search.api.SearchFieldKey;
+import edu.unc.lib.boxc.search.api.facets.CutoffFacet;
+import edu.unc.lib.boxc.search.api.models.ContentObjectRecord;
+import edu.unc.lib.boxc.search.api.requests.SearchRequest;
+import edu.unc.lib.boxc.search.api.requests.SearchState;
+import edu.unc.lib.boxc.search.api.requests.SimpleIdRequest;
+import edu.unc.lib.boxc.search.solr.filters.QueryFilterFactory;
+import edu.unc.lib.boxc.search.solr.services.SolrSearchService;
+import org.apache.commons.csv.CSVPrinter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Objects;
+
+import static edu.unc.lib.boxc.model.api.ResourceType.Work;
+import static edu.unc.lib.boxc.web.services.utils.CsvUtil.cleanupCsv;
+import static edu.unc.lib.boxc.web.services.utils.CsvUtil.createNewCsvPrinter;
+import static java.util.Arrays.asList;
+
+public class BulkRefIdCsvExporter {
+    private static final Logger log = LoggerFactory.getLogger(BulkRefIdCsvExporter.class);
+    private static final int DEFAULT_PAGE_SIZE = 10000;
+    public static final String PID_HEADER = "pid";
+    public static final String REF_ID_HEADER = "refId";
+    public static final String HOOK_ID_HEADER = "hookId";
+    public static final String TITLE_HEADER = "title";
+    public static final String[] CSV_HEADERS = new String[] {PID_HEADER, REF_ID_HEADER, HOOK_ID_HEADER, TITLE_HEADER};
```

Comment:
Would probably be best to name this EXPORT_CSV_HEADERS, similar to how to renamed the other one IMPORT_CSV_HEADERS in this PR